main.go

```diff
@@ -204,6 +204,7 @@ func main() {
 	// Set the API Server QPS and Burst
 	kubeConfig.QPS = config.DefaultAPIServerQPS
 	kubeConfig.Burst = config.DefaultAPIServerBurst
+	kubeConfig.UserAgent = fmt.Sprintf("%s/%s", config.ControllerName, version.GitVersion)
```

Comment:
Can we use the same AppName at https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/master/pkg/aws/ec2/api/wrapper.go#L42? The name is used for EC2 dashboard as client user. We should use the same name if possible.

Reply:
Updated this to use `ec2API.AppName`